src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -98,14 +98,14 @@ public static Name parseName(String name) throws ParseException {
      * @throws ParseException if the given {@code phone} is invalid.
      */
     public static Phone parsePhone(String phone) throws ParseException {
-        if (phone != null) {
-            String trimmedPhone = phone.trim();
-            if (!Phone.isValidPhone(trimmedPhone)) {
-                throw new ParseException(Phone.MESSAGE_CONSTRAINTS);
-            }
-            return new Phone(trimmedPhone);
+        if (phone.isBlank()) {
+            return new Phone(null);
```

Comment:
Would it be more appropriate to just return null instead of Phone(null)
```suggestion
            return null;
```

Reply:
After looking at how edit works, it might be better to leave it as new *(null)
Interview date should be changed to be new InterviewDate(null) instead